README.md

```diff
@@ -375,6 +376,19 @@ Give your images a shadow and rounded corners.
 
 ---
 
+## [Media Grid](https://github.com/zremboldt/obsidian-media-grid)
+
+by [@zremboldt](https://github.com/zremboldt)
+
+Add more meaning and visual interest to your documents by easily grouping sets of images, videos, or even audio tracks into grid layouts.  
+I've linked to my repo for a more thorough explanation of custom settings, requirements, how it works, etc.
```

Comment:
You are welcome to write a longer explanation if you deem it necessary. In the end you can add something like:

```markdown
For more information on how to use this snippet, [see it's original repo](https://github.com/zremboldt/obsidian-media-grid).
```

or something similar.

Reply:
Sounds good. I'll try to get back to this tonight. Thank you!